appsdk/sdk_test.go

```diff
@@ -415,15 +405,15 @@ func TestSetLoggingTargetRemote(t *testing.T) {
 	assert.Equal(t, "http://logs:8080/api/v1/logs", result, "File path is incorrect")
 }
 func TestInitializeClientsAll(t *testing.T) {
-	clients := make(map[string]common.ClientInfo)
-	clients[common.CoreDataClientName] = common.ClientInfo{}
-	clients[common.NotificationsClientName] = common.ClientInfo{}
-	clients[common.CoreCommandClientName] = common.ClientInfo{}
+	clientMap := make(map[string]common.ClientInfo)
```

Comment:
Don't use type in the variable name. Leave as `clients`

Reply:
`clients` collides with a package name and generates a linter warning.